css/_language-picker.scss

```diff
@@ -1,16 +1,24 @@
 .language-picker {
-    background-color: rgba(255, 255, 255, 0.6);
+    background-color: rgba(255, 255, 255, 0.84);
     color: #090C0E;
     position: relative;
-    padding: 12px;
+    padding: 12px 8px;
     transition: background-color var(--transition-out), color var(--transition-out);
     min-width: 216px;
 
     display: -ms-inline-grid; display: inline-grid;
```

Comment:
De chevron staat nu met `position: absolute` gepositioneerd. Als je die static maakt, moet die op basis van het grid kunnen uitlijnen?

Reply:
Als je toch een grid gebruikt, misschien de witruimte met behulp van `gap` creëren?

Reply:
Dit is denk ik iets dat ik heb geintroduceerd en ik wil daar best even naar kijken @mellewynia en @mikecodo :)

@hdv ik probeerde hem op static te zetten, maar de chevron wordt aangemaakt via een pseudo-element, en ik liep tegen wat issues aan. Ik denk dat als ik dit op zou pakken ik hier een svg van zou maken (een `chevron-down.svg` is al aanwezig in deze repo) en dan die absolute positionering er af zou halen.